BackendTaskPilot/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
```

Comment:
for the constant url, write following:

{
  "ExternalUrls": {
    "CnbCurrenciesUrl": "https://api.cnb.cz/cnbapi/exrates/daily"
  }
}